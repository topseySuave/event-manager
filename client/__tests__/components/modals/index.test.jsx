import React from 'react';
import { mount, shallow } from 'enzyme';
import Modals from
  '../../../src/components/modals/';

describe('Modals component', () => {
  const wrapper = shallow(<Modals />);
  test('component should mount', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
