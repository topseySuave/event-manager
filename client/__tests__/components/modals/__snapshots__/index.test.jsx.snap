// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modals component component should mount 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Modals />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="modal modal-fixed-footer"
          id="add_event_modal"
        >
          <div
            className="modal-content"
          >
            <h4>
              <span>
                Edit
              </span>
               Event
            </h4>
            <Connect(EditEventModal) />
          </div>
          <div
            className="modal-footer"
          >
            <a
              className="modal-action modal-close waves-effect btn-flat"
              href="#_=_"
            >
              Cancel
            </a>
          </div>
        </div>,
        <CenterModal />,
        <Connect(SearchModal) />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="modal-content"
            >
              <h4>
                <span>
                  Edit
                </span>
                 Event
              </h4>
              <Connect(EditEventModal) />
            </div>,
            <div
              className="modal-footer"
            >
              <a
                className="modal-action modal-close waves-effect btn-flat"
                href="#_=_"
              >
                Cancel
              </a>
            </div>,
          ],
          "className": "modal modal-fixed-footer",
          "id": "add_event_modal",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h4>
                  <span>
                    Edit
                  </span>
                   Event
                </h4>,
                <Connect(EditEventModal) />,
              ],
              "className": "modal-content",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <span>
                      Edit
                    </span>,
                    " Event",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Edit",
                    },
                    "ref": null,
                    "rendered": "Edit",
                    "type": "span",
                  },
                  " Event",
                ],
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                className="modal-action modal-close waves-effect btn-flat"
                href="#_=_"
              >
                Cancel
              </a>,
              "className": "modal-footer",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Cancel",
                "className": "modal-action modal-close waves-effect btn-flat",
                "href": "#_=_",
              },
              "ref": null,
              "rendered": "Cancel",
              "type": "a",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="modal modal-fixed-footer"
            id="add_event_modal"
          >
            <div
              className="modal-content"
            >
              <h4>
                <span>
                  Edit
                </span>
                 Event
              </h4>
              <Connect(EditEventModal) />
            </div>
            <div
              className="modal-footer"
            >
              <a
                className="modal-action modal-close waves-effect btn-flat"
                href="#_=_"
              >
                Cancel
              </a>
            </div>
          </div>,
          <CenterModal />,
          <Connect(SearchModal) />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="modal-content"
              >
                <h4>
                  <span>
                    Edit
                  </span>
                   Event
                </h4>
                <Connect(EditEventModal) />
              </div>,
              <div
                className="modal-footer"
              >
                <a
                  className="modal-action modal-close waves-effect btn-flat"
                  href="#_=_"
                >
                  Cancel
                </a>
              </div>,
            ],
            "className": "modal modal-fixed-footer",
            "id": "add_event_modal",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h4>
                    <span>
                      Edit
                    </span>
                     Event
                  </h4>,
                  <Connect(EditEventModal) />,
                ],
                "className": "modal-content",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <span>
                        Edit
                      </span>,
                      " Event",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Edit",
                      },
                      "ref": null,
                      "rendered": "Edit",
                      "type": "span",
                    },
                    " Event",
                  ],
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="modal-action modal-close waves-effect btn-flat"
                  href="#_=_"
                >
                  Cancel
                </a>,
                "className": "modal-footer",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Cancel",
                  "className": "modal-action modal-close waves-effect btn-flat",
                  "href": "#_=_",
                },
                "ref": null,
                "rendered": "Cancel",
                "type": "a",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
